Common/Data/FileHelper.swift

```diff
@@ -302,6 +326,8 @@ public class FileHelper {
     }
 
     public static func updateDatabase(contentsOf parentFolder: File, using contents: [String: Any]) -> Future<Void, SCError> {
+        //TODO: Rethink this
+
```

Comment:
?

Reply:
I unfortunately don't remember what I meant there. But I made some changes that should cover what i initially thought.